tests/model/operation/transform/move.js

```diff
@@ -224,21 +224,6 @@ describe( 'transform', () => {
 					'</blockQuote>'
 				);
 			} );
-
-			it( 'intersecting wrap', () => {
```

Comment:
Why you removed that test?

Reply:
I removed it, because wrap was on text, not element. We decided to remove those cases.

Reply:
👍